lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -502,8 +516,14 @@ Function *PatchEntryPointMutate::lowerCpsFunction(Function *func, ArrayRef<Type
 
   AttributeList oldAttrs = func->getAttributes();
   SmallVector<AttributeSet, 8> argAttrs;
-  for (unsigned idx = 0; idx != userDataTys.size(); ++idx)
+  bool hasLocalInvocationIdArg = argNames.back() == "LocalInvocationId";
```

Comment:
Depending on the name of an argument seems rather brittle. Is there any better way how we can find out if we have a local invocation id (like the condition for `cpsShaderInputs` below)?

Reply:
I just add one boolean argument for it.